app/views/layouts/application.html.erb

```diff
@@ -17,7 +17,22 @@
   <head>
     <meta charset="UTF-8" />
     <title>
-      Sequencescape : <%= controller.controller_name.capitalize %> - <%= @page_name || controller.action_name %>
+      <% # Define a hash of environments
+        environments = {
+          cucumber: '🥒',
+          development: '🚧',
+          production: '🌎',
+          profile: '⏱️',
+          staging: '🚀',
+          test: '🧪',
+          training: '🎓',
+        }
+
+        # Get an emoji for the current environment
+        current_environment = Rails.env.to_sym
+        emoji = environments[current_environment] || '❓' 
+      %>
+      <%= "#{emoji}:" unless Rails.env.production? %> Sequencescape : <%= controller.controller_name.capitalize %> - <%= @page_name || controller.action_name %>
     </title>
```

Comment:
It is changing the title text. It is putting an emoji between the favicon and the existing title. I think a neater way of doing this would be changing the favicon (logo) colour. 

When there are too many tabs open, it looks like a clutter and emoji (proposed start of title) goes hidden.

Reply:
- These are matching the navigation bar colours. 
- They remove the clutter from browser tab buttons. 
- They have simplicity compared to the emojis.
- They use the top field above twirling lines as the environment colour.
- They fill the space between white twirling lines same as the top field colour, following the same design as Production icon, to provide contrast and put the purpose in the middle as well.
- For Dev and UAT icons only, a single colour could be considered if it helps distinguishing in browser tab buttons but I think it is not necessary and against the original icon design.

Additional: Login page's login panel image could use the same pattern.